java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -161,4 +171,23 @@ public void testPerTaskEnvVarsOverwritePerJobEnvVars() {
       Ray.shutdown();
     }
   }
+
+  public void testJarsInActor() {
+    try {
+      Ray.init();
+      final RuntimeEnv runtimeEnv =
+          new RuntimeEnv.Builder()
+              .addJars(
+                  ImmutableList.of(
+                      "https://ray-mobius-us.oss-us-west-1.aliyuncs.com/ci/linux/actor-observer-tmp.jar"))
```

Comment:
Can we avoid depending on the external server?
We can package the jar at runtime and use a simple python http server to serve it.

Reply:
I believe a prepared package is easier for test. It's no need to do heavy packaging and serving when running a test.

Reply:
I found that there are a lot existing tests which depend on http source, e.g. https://github.com/ray-project/ray/blob/master/python/ray/tests/test_runtime_env_working_dir_remote_uri.py#L15. So, it's ok to test like this. But I think we can try to use github.com instead of the oss of aliyun. We should avoid using multiple external sources for the stability of UT.

Reply:
In addition, can you also add test for ".zip" files? You can also add two files at the same time to ensure we could support multiple packages.

Reply:
> In addition, can you also add test for ".zip" files? You can also add two files at the same time to ensure we could support multiple packages.

Good suggestion. Added.

Reply:
@SongGuyang An issue was created for that https://github.com/ray-project/ray/issues/24534